include/onnxruntime/core/common/optional.h

```diff
@@ -3,8 +3,11 @@
 
 #pragma once
 
+#define optional_CONFIG_SELECT_OPTIONAL optional_OPTIONAL_NONSTD
```

Comment:
it might be helpful to have a comment explaining what this does and why we need it